.github/workflows/ruby_head_specs.yml

```
@@ -31,6 +31,7 @@ jobs:
           ruby-version: head
           bundler: latest
           bundler-cache: true
+          debug: true
```

actionlint (1.7.4)

input "debug" is not defined in action "ruby/setup-ruby@v1". available inputs are "bundler", "bundler-cache", "cache-version", "ruby-version", "rubygems", "self-hosted", "windows-toolchain", "working-directory"

(action)